src/main/java/emis.task/Task.java

```diff
@@ -0,0 +1,48 @@
+package main.java.emis.task;
+import main.java.emis.Ui;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task(boolean isDone, String description) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+        Ui.showLine();
+        System.out.println("\tNice! I've marked this task as done:");
+        System.out.println("\t\t[" + this.getStatusIcon() + "] " + this.description);
```

Comment:
I wonder if these lines can be in Ui class as well since they are showing things to the user?